introduction.md

```diff
@@ -6,41 +6,170 @@ layout: documentation
+
+Bindings are software packages that are installed by the user in openHAB. 
+The main purpose of Bindings are to establish the connection between your device and your [Thing]({{base}}/concepts/things). 
+Bindings communicate with your device and translate all commands to and from openHAB between your device and your [Thing]({{base}}/concepts/things).
+
+Bindings are provided at the [Add-on section](https://www.openhab.org/addons/) of the [openHAB website](https://www.openhab.org). 
+Here you will find a searchable list of several hundred bindings to support as many devices as possible. 
+New bindings are regularly added as developers integrate more devices into openHAB.
+
+For each binding, detailed instructions and examples are provided that include guidance on configuration (if any) of the binding itself, the definition of [Things]({{base}}/concepts/things) supported by this binding and the Channels these [Things]({{base}}/concepts/things) provide. 
+In most cases, the description also contains a fully worked out example that includes a definition of [Things]({{base}}/concepts/things) and its Channels, [Items]({{base}}/concepts/items) linked to those Channels and the use of these [Items]({{base}}/concepts/items) in a sitemap.
+
+## While you are getting started
+
+To all newcomers: please read the section [New User Tutorial]({{base}}/tutorial). 
+That section provides you with a step-by-step instructions for your first easy setup. 
+
+OpenHAB runs on most popular platforms such as Linux, Windows and MacOS and on almost any hardware ranging from Raspberry Pis to desktop computers and server PCs. 
+You can find specific installation instructions for these and other platforms in the [Installation Overview]({{base}}/installation/index.html) article. 
+If you have a strong preference towards a particular platform, then that platform is probably your best choice.
+
+You can install openHAB on your desktop computer for evaluation purposes if you already have any of these systems available for use, but we recommend using a dedicated system in the long run. 
+If you feel serious about home automation it may be better to start with a dedicated system right away.
+
+For anyone undecided: many people find that the simplest way to experiment with openHAB is to get a [Raspberry Pi (Version 2  or 3)](https://raspberrypi.org) and install [openHABian]({{base}}/installation/openhabian.html); a "hassle-free openHAB setup". 
+While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB home automation system easily capable of automating your entire home.
+However, it is worth noting two main limitations for Raspberries. 
+They are limited in the RAM (memory) and may not perform well when additional database and data visaulization programs are installed.
+Running Raspberries from SD-cards only may result in system instabilities as these memory cards can degrade quickly under openHAB's use conditions. 
```

Comment:
This is not true. Yes it is wise to think about backups and yes there have been users with bad luck. This is not the general case, otherwise someone needs to explain to me why the 11th iteration of the Raspberry Pi is still entrusted with an SD card 😉

Reply:
> This is not true. Yes it is wise to think about backups and yes there have been users with bad luck. This is not the general case, otherwise someone needs to explain to me why the 11th iteration of the Raspberry Pi is still entrusted with an SD card 😉

Because Pis are build to learn programming. They are not meant to be 24x7 servers.
I know you don't believe in it and yes it is not a compelling result and it depends on a number of factors if it happens (too difficult to discuss in that here or in any introductory article), but for sure there's way too many people to have suffered from this than for this to be "not true" either. Also note that "use conditions" also include power outages which are frequent in HA and another way to crash SD cards.
I think this general warning covers both and should therefore remain in the docs as it stands now.
Omitting it would frustrate a number of people when their system crashes and they would attribute that to openHAB, not their Pi.
And yes, the install instruction should be reworked with a more extensive explanation, too.

Reply:
I understand that the discussion can be more nuanced and complicated. This part is the very first introduction, we simply cannot overload it with too many technical details. This intro is also deliberately written to connect with real novices. Anyone with more expertise will just glance at it sigh and move on. I am agreeing with @mstormi and suggest to keep as is